### PR TITLE
display profile image in form if present and add option to remove it

### PR DESCRIPTION
**Pull request by lislis**
#### Is this solving an issue already in our backlog? If so, please mention which one
#603

#### Do you have any open Questions you need to get answered?
Not sure how the option to delete the image should work.
I check a checkbox that then calls a remove image method somewhere?

#### Please outline the areas that should be tested, if there are any:
n/a

### CODE REVIEW

**Comment by lislis** [comment]
First step: display profile image if one is present

screenshot-2018-2-20 rorganize it find a rails girls study group

**Comment by lislis** [comment]
Now looks like this:
screenshot-2018-2-27 rorganize it find a rails girls study group

---


screenshot-2018-2-27 rorganize it find a rails girls study group 1

File: spec/features/group_sub_pages_spec.rb [hunk 1]
```diff
@@ -22,7 +22,6 @@
     scenario 'Find groups searching for something' do
       visit_group_main_page
       click_link 'Groups searching'
-      save_and_open_page
```
Comment (vsmart) [inline spec/features/group_sub_pages_spec.rb]: 👋 Did you mean to remove this?

Comment (lislis) [reply spec/features/group_sub_pages_spec.rb]: yes! It's a leftover debug method. It opens the rendered html page in the bowser so you can see what the test 'sees'.

Comment (vsmart) [reply spec/features/group_sub_pages_spec.rb]: Cool 🆗

File: spec/features/edit_person_spec.rb [hunk 2]
```diff
@@ -41,15 +41,33 @@
...
 
     scenario 'with incomplete link' do
-      fill_in "Website", with: 'malweene.com'
+      fill_in "Your website", with: 'malweene.com'
       click_button 'Save'
       expect(page).to have_content 'http://malweene.com'
     end
   end
+
+
```
Comment (sareg0) [inline spec/features/edit_person_spec.rb]: perhaps we could get rid of the extra empty lines here?

**Comment by sareg0** [comment]
supey nice! I left a very minor comment. Happy to merge after that.

**Pull request merged**